src/navigate-to-subcategory.spec.ts

```diff
@@ -0,0 +1,32 @@
+import {browser, by, ExpectedConditions} from "protractor";
+import { MainPage } from "./page-object/main-page";
+
+const mainPage = new MainPage();
+describe('Navigate', () => {
+    beforeAll(async() => {
+        await browser.get('http://automationpractice.com/');
+    });
+
+    it('to Dresses subcategory from Women page', async () => {
+        await mainPage.changeCategory('Women');
+        await browser.wait(ExpectedConditions.visibilityOf(mainPage.searchCounter), 5000); //I wanted to find some unique element after the previous step, but the site is stupid))
```

Comment:
please store 5000 ms in some constant class. You can ask Yarik for help

Reply:
fixed